press/utils/__init__.py

```diff
@@ -543,3 +545,15 @@ def poly_get_doctype(doctypes, name):
 		if frappe.db.exists(doctype, name):
 			return doctype
 	return doctypes[-1]
+
+
+def reconnect_on_failure():
+	@wrapt.decorator
```

Comment:
Where are you getting `wrapt` from? :laughing: It's not defined in req.txt or setup.py